docs/troubleshooting/harvester.md

```diff
@@ -4,7 +4,37 @@ sidebar_label: Harvester
 title: ""
 ---
 
-## Generate a support bundle
+## Fail to deploy multi-node Cluster due to incorrect HTTP Proxy Setting
+
+### Configure HTTP Proxy during Harvester Installation
+
+In some environments, you configure [HTTP Proxy](../airgap/#configure-an-http-proxy-during-installation) during Harvester installation.
+
+### One NODE becomes unavailable
+
+The issue you may encounter:
+
+```
+The first NODE is installed successfully.
+
+The second NODE is installed successfully.
+
+The third NODE is installed successfully.
+
+Then the second NODE changes to Unavialable state and cannot recover automatically.
+```
+
+### Solution
+
+When the `NODEs` in the cluster are not using the HTTP Proxy to communicate with each other, after the first `NODE` is installed successfully, you need to login into `Harvester GUI` to config [no-proxy](../airgap.md#configure-an-http-proxy-in-harvester-settings) against the CIDR used by those `NODEs`.
+
+For example, one cluster assigns IP from CIDR `172.26.50.128/27` to `NODEs` via DHCP/static setting, please add this CIDR into `no-proxy`.
```

Comment:
```suggestion
For example, if a cluster assigns IPs from CIDR `172.26.50.128/27` to nodes via DHCP/static setting, please add this CIDR to `no-proxy`.
```

Reply:
Not sure if it means `if`. Please double check

Reply:
I changed to `your cluster`, it fits more to the context.

The `login into` should be correct, it means user inputs username and password in the login page.